### ISSUE TITLE
PHP 7 compatibility

### ISSUE THREAD

**Issue by ghost** [pbearne/wp-author-avatars#12]
Hi Paul,

Moving the discussion here from https://wordpress.org/support/topic/deprecated-php-4-style-constructors.

With the new version there still are some problems:

```
Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; AuthorAvatars has a deprecated constructor in /wp-content/plugins/author-avatars/lib/AuthorAvatars.class.php on line 27

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; AuthorAvatarsEditorButton has a deprecated constructor in /wp-content/plugins/author-avatars/lib/AuthorAvatarsEditorButton.class.php on line 7
```

**Comment by mudtar** [comment]
It looks like the last of the necessary fixes were made in be03956f04fa9f4e02a9450791239c6de44c8ec1 and e5e91b0e5e838350baae137135e18062afb04cbe. @pbearne: Do you expect to be rolling out a new version via the WP plugin repo anytime soon?

**Comment by pbearne** [comment]
yes I will get to this ASP

*Paul Bearne*
*Freelance Senior Full-Stack WordPress Developer*
*Cell: +1 647 780 7594*
*Home Office: +1 416 792 8894*
*E-mail: **paul@bearne.com* <pbearne@gmail.com>
*@pbearne <http://twitter.com/pbearne> *or +pbearne
<https://plus.google.com/u/0/100456877588970933323>
